src/features/users/user-controller.ts

```diff
@@ -115,24 +116,33 @@ class UserController {
         if (universities.length > 1) {
             logger.error(`Multiple universities found ${universities.length}`);
         }
-        university = universities[0];
+        const university = universities[0];
 
 
         if (university.student_email_domain === emailDomain) {
-            logger.info('User is student');
+            // Database field
```

Comment:
Logger statements could still be useful, no?

Reply:
Seems like bloat when it is in the response